tornado/auth.py

```diff
@@ -899,14 +916,18 @@ async def get(self):
 
            The ``callback`` argument was removed. Use the returned awaitable object instead.
         """  # noqa: E501
-        handler = cast(RequestHandler, self)
+
+        if not client_id:
+            client_id = self.get_google_oauth_settings()['key'],
```

Comment:
The trailing commas here and the next line are a bug (caught by mypy). Fix coming in #3180